src/ksym.rs

```diff
@@ -65,17 +62,20 @@ impl KSymResolver {
                     continue
                 }
                 let name = String::from(func);
-                self.syms.push(Ksym { addr, name });
+                syms.push(Ksym { addr, name });
```

Comment:
we should eventually capture (and use ?) module name. Not blocking right now, of course.

Reply:
Makes sense. Will do that once we have use for it.